src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,88 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    static final int SIZE_ARRAY = 3;
+    static final int SIZE_SPLIT_DATA_ARRAY = 4;
+    static final int INDEX_DATE = 0;
+    static final int INDEX_DAY = 0;
+    static final int INDEX_MONTH = 1;
+    static final int INDEX_YEAR = 2;
+    static final int INDEX_NAME = 1;
+    static final int INDEX_HOURS = 2;
+    static final int INDEX_MONEY = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        final String[] dateFromSplit = dateFrom.split("\\.");
+        final String[] dateToSplit = dateTo.split("\\.");
```

Comment:
It is better to use the LocalDate
```suggestion
        LocalDate parsedDateFrom = LocalDate.parse(dateFrom, formatter);
        LocalDate parsedDateTo = LocalDate.parse(dateTo, formatter);
```